### ISSUE TITLE
changing height after element was out-of-screen

### ISSUE THREAD

**Issue by luxferresum** [runspired/smoke-and-mirrors#153]
I have a `vertical-collection` where the hight of the items can change between two states with different hights.

I use `alwaysRemeasure=true` and `defaultHeight=50`, which is the smaller height.
Now when I change an item to the `tall` state and scroll it off-screen SAM sets `min-height: 200px;`. The `tall` state actually *is* `200px` tall. But when I scroll the item back into the viewport the `min-height` remains `200px`, and I can't change the state back to the `small` state.

So basically the items can only get larger and larger. 😢 

I think SAM should set the `min-height` back to the `defaultHeight` when an items gets back into the viewport, and then again remeasure when it leaves the viewport again.

**Comment by sglanzer** [comment]
I just ran across the same scenario - 👍 for a fix on this.  I'd be happy to submit a PR given some direction toward the right area of the code to look at.

**Comment by luxferresum** [comment]
I'm pretty sure we just have to insert something like `this.element.style.minHeight  = this.get('defaultHeight')` in the right place in the `vertical-item` component. But this needs further investigation. 

@runspired would you accept a PR? Then I could maybe spend some time investigating this.

**Comment by runspired** [comment]
Should be fixed by updating to `@html-next/vertical-collection` (now re-exported by this lib)
